demo/src/main/AndroidManifest.xml

```diff
@@ -43,6 +42,7 @@
             android:value="${MAPS_API_KEY}" />
 
         <activity
+            android:exported="true"
```

Comment:
Also curious about this change

Reply:
This PR targets Android 12+, and since then it is compulsory to specify the `android:exported` value. See official statement [here](https://developer.android.com/about/versions/12/behavior-changes-12#exported).